### ISSUE TITLE
Uploads automatic cleanup

### ISSUE THREAD

**Issue by Moondownr** [slskd/slskd#287]
After a week or so of not checking on the instance of slskd, the list of uploads is huge and daunting to clean manually. The easy solution is to restart slskd, however it may not always be ideal solution for the user.
Would you consider adding an option to autodelete completed/errored uploads after a set period and/or buttons to manually clean up all finished/errored as in nicotine?

**Comment by jpdillingham** [comment]
Absolutely!  A button to clear all completed transfers is a good place to start, and relatively easy to implement.

Automatically clearing transfers after a period of time would take some more doing, and I'd prefer to hold off until I can get a feature in place to log completed transfers to a database, as currently there's no record of a transfer having taken place once it is removed.
